recipes-core/aziot-edged/aziot-edged_1.4.20.bb

```diff
@@ -7,8 +7,8 @@ inherit cargo
 
 # how to get aziot-edged could be as easy as but default to a git checkout:
 # SRC_URI += "crate://crates.io/aziot-edged/0.1.0"
-SRC_URI += "git://github.com/Azure/iotedge;protocol=https;nobranch=1"
-SRCREV = "b4c7dc64765c1b33aa563ff0e7a317516b9fce00"
+SRC_URI += "git://github.com/Azure/iotedge.git;protocol=https;nobranch=1"
```

Comment:
.git file will be make any difference?

Reply:
Honestly not sure. That's what `cargo bitbake` generated, and I would guess it's fine. I left it in there because I'd like to deviate from the generated code as _little_ as possible. If this works, then the diff from one release to the next will be that much smaller in the future.